prowler/providers/aws/services/elbv2/elbv2_service.py

```diff
@@ -8,20 +8,18 @@
 from prowler.providers.aws.lib.service.service import AWSService
 
 
-################### ELBv2
 class ELBv2(AWSService):
     def __init__(self, provider):
         # Call AWSService's __init__
         super().__init__(__class__.__name__, provider)
-        self.loadbalancersv2 = []
-        self.__threading_call__(self.__describe_load_balancers__)
-        self.listeners = []
-        self.__threading_call__(self.__describe_listeners__)
-        self.__threading_call__(self.__describe_load_balancer_attributes__)
-        self.__threading_call__(self.__describe_rules__)
-        self.__describe_tags__()
-
-    def __describe_load_balancers__(self, regional_client):
+        self.loadbalancersv2 = {}
+        self.__threading_call__(self._describe_load_balancers)
+        self.__threading_call__(self._describe_listeners)
+        self.__threading_call__(self._describe_load_balancer_attributes)
+        self.__threading_call__(self._describe_rules)
+        self._describe_tags()
```

Comment:
Please, could you use a thread for each load balancer using?
```suggestion
        self.__threading_call__(self._describe_load_balancer_attributes, self.loadbalancers)
```